pkg/cluster/sync.go

```diff
@@ -500,6 +500,7 @@ func (c *Cluster) syncSecrets() error {
 				c.logger.Warningf("secret %q does not contain the role %q", secretSpec.Name, secretUsername)
 				continue
 			}
+			c.Secrets[secret.UID] = secret
```

Comment:
How is this related to the infrastructure roles?

Reply:
moved to #1091